problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,52 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob
+		    Use set notations, vectors, or linear combinations of vectors to describe the following sets.
+		    \begin{enumerate}
+		        \item The set of vectors with rational components in $\R^2$.
+		        \item The set of vectors with irrational components in $\R^2$.
+		        \item The set that contains the empty set.
```

Comment:
This is not good...Maybe, "The set of vectors consisting only of [1,2]"

Reply:
My intention was to distinguish a collection of sets from a set of vectors, an object confusion mistake that students had in the past. Is this too indirect?

Reply:
Yes. Empty sets are harder than non-empty sets. You could do something like, "Let $P(\vec x) = {\vec x, -\vec x}". The set ${P(e_1), P(e_2), P(e_3)$